practice-two/src/css/style.css

```diff
@@ -9,12 +9,16 @@
   --font-size-m: 24px;
 }
 
-.registration-form {
+.border {
```

Comment:
Class `border` should not contain margin property. Try to style on .registration-form and .list-user directly. Or rename border to suitable name